stacks/stack.go

```diff
@@ -66,7 +66,9 @@ type Stack struct {
 
 // SetStackName - sets the stackname with struct
 func (s *Stack) SetStackName() {
-	s.Stackname = fmt.Sprintf("%s-%s", *s.Project, s.Name)
+	if s.Stackname == "" {
```

Comment:
Please see comment for previous changes. In fact we have two stack names: the name of the stack in terms of the key in ``config.yml`` and the name of the stack in CloudFormation context. As ``s.Stackname`` already exists and used for AWS API calls, I would suggest to keep it and let the user overwrite it by the now renamed "Name" key. Otherwise we might refactor ``s.Stackname`` to something like ``s.CfStackname`` to emphasize the difference.